src/components/ButtonPanel.js

```diff
@@ -4,19 +4,29 @@ import Button from './Button';
 
 const buttonNames = [
   ['AC', '+/-', '%', '÷'],
-  ['7', '8', '9', 'X'],
+  ['7', '8', '9', 'x'],
   ['4', '5', '6', '-'],
   ['1', '2', '3', '+'],
   ['0', '.', '='],
 ];
 
+const orangeButtons = [
+  '÷', 'x', '-', '+',
+];
+
 const ButtonPanel = () => (
-  <div>
+  <div className="button-panel">
     {buttonNames.map(group => (
       <div key={uuidv4()} className="board-row">
-        {group.map(element => (
-          <Button key={uuidv4()} name={element} />
-        ))}
+        {group.map(element => {
+          if (orangeButtons.includes(element)) {
+            return <Button key={uuidv4()} name={element} wide={false} />;
+          }
+          if (element === '0') {
+            return <Button key={uuidv4()} name={element} color="gray" wide />;
+          }
+          return <Button key={uuidv4()} name={element} color="gray" wide={false} />;
+        })}
```

Comment:
OPTIONAL: Are the keys going to be generated on every render? Remember that every time a component's key changes [React will create a new component instance rather than update the current one](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key:~:text=React%20will%20create%20a%20new%20component%20instance%20rather%20than%20update%20the%20current%20one.).

Reply:
Thank you for the feedback.